cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -678,10 +704,8 @@ private void getPartitions(@Nullable PartitionFilter filter, PartitionConsumer c
         try {
           key = parseRowKey(row.getRow(), partitioning);
         } catch (IllegalArgumentException e) {
-          if (!ignoreInvalidRowsSilently) {
```

Comment:
I don't know why it was here, but it was always false. It's never set to true, unless someone's application is modifying it via reflection...